src/types/externalAPI.ts

```diff
@@ -24,4 +24,5 @@ export interface ExternalAPI {
     createListener(event: 'desktop-sources-modal-request', listener: () => void): () => void;
     createListener(event: 'calls-widget-open-thread', listener: (threadID: string) => void): () => void;
     createListener(event: 'calls-widget-open-stop-recording-modal', listener: (channelID: string) => void): () => void;
+    createListener(event: 'calls-widget-open-user-settings', listener: () => void): () => void;
```

Comment:
For my understanding, how come this file does not use the constants in `src/common/communication.ts`?

Reply:
I'll defer that to @devinbinnie

Reply:
Because it's a types file and TypeScript does not recognize those constants as the type of `const` where the value is that string. In order to make the typing actually work, you need to put it the actual string as the type, not the value.